spec/forms/candidate_interface/gcse_qualification_details_form_spec.rb

```diff
@@ -118,6 +118,24 @@
           expect(form.errors[:grade]).to include('Enter a real grade')
         end
       end
+
+      it 'returns validation error if award year is after 1988' do
+        form.award_year = '2012'
+        form.validate(:award_year)
+
+        expect(form.errors[:award_year]).to include('Enter a year before 1989')
+      end
+
+      it 'returns no error if award year is valid' do
+        valid_years = (1951..1988)
```

Comment:
i had no idea you could do this. I've always used  [*1951..1988]. That's cool to know 👍